src/components/App/App.jsx

```diff
@@ -33,6 +35,21 @@ function App() {
     }
   })
 
+  function handleStart() {
+    setIsActive(true)
+    const currentDateObj = new Date()
+    const endDateObj = new Date()
+
+    lengthLong
```

Comment:
You could consider renaming this variable (and others) using "duration" instead of "length", because _duration_ would be more precise for time units => **durationLong**. _length_ might be confused with distance.

Reply:
Great idea! 🙏